wandb/xgboost/__init__.py

```diff
@@ -0,0 +1,20 @@
+'''W&B callback for xgboost
+
+Really simple callback to get logging for each tree
+
+Example usage:
+
+param_list = [("eta", 0.08), ("max_depth", 6), ("subsample", 0.8), ("colsample_bytree", 0.8), ("objective", "multi:softmax"), ("eval_metric", "merror"), ("alpha", 8), ("lambda", 2), ("num_class", 10)]
+config.update(dict(param_list))
+bst = xgb.train(param_list, d_train, callbacks=[wandb_callback()])
+'''
+
+import wandb
+import xgboost
+
+def wandb_callback():
```

Comment:
It probably makes sense to name this WandbCallback to be consistent with the rest of the callbacks.

Reply:
I don't know about this: WandbCallback is a class - in this case I'm defining a function.  Do we really and to uppercase a class?  Seems kind of wild.